Catalog/Shared/Toggles/ToggleStyles.swift

```diff
@@ -46,12 +47,28 @@ struct SunMoonToggleSample: PropSampleable {
     }
 }
 
+struct GravityToggleSample: PropSampleable {
+    let name = "gravity toggle style"
+    let notes: String? = ".toggleStyle(.gravity)"
+    let source = "Custom"
+    let keywords = "toggle custom props animated gravity"
```

Comment:
'props" here was "third party" which referenced the sun/moon toggle above which is literally a 3rd party SPM imported into the catalog. Let's keep it consistent when we're using our props. Keep an eye on mine, too, as I'm likely to do this, too. :D

Reply:
Oh right! I see. We should also change the source to be "Props" and not "Custom" then, to be consistent with others?

I've used your changes and applied to that other PR, but we'll check it out after this one ;)